alert-common/src/main/java/com/synopsys/integration/alert/common/workflow/formatter/SummaryMessageContentFormatter.java

```diff
@@ -69,11 +69,14 @@ public SummaryMessageContentFormatter(MessageOperationCombiner messageOperationC
         List<MessageContentGroup> newGroups = new ArrayList<>();
         for (ProviderMessageContent message : collapsedMessages) {
             ProviderMessageContent summarizedMessage = summarizeMessageContent(message);
-            newGroups
-                .stream()
-                .filter(group -> group.applies(summarizedMessage))
-                .findAny()
-                .ifPresentOrElse(group -> group.add(summarizedMessage), () -> newGroups.add(MessageContentGroup.singleton(summarizedMessage)));
+
+            if (summarizedMessage.getAction().isPresent() || !summarizedMessage.getComponentItems().isEmpty()) {
```

Comment:
Do we know why MessageContentFormatter.filterEmptyContent() doesn't take care of this?